lib/literature.ex

```diff
@@ -705,4 +706,42 @@ defmodule Literature do
   end
 
   defp parse_tags(data), do: data
+
+  def sort_tag_posts(post_ids, tag_id) do
+    existing_post_ids =
```

Comment:
@rcmonsayac why do we need to check exiting tags here ? Also what happens if a tag is created or deleted with the order ?

Reply:
`why do we need to check exiting tags here`
 - since I am using `Repo.insert_all` with `conflict_target` to bulk update,  I wanted to check first if the TagPost association records exist so I do not accidentally insert records.
 
`Also what happens if a tag is created or deleted with the order`
- I was thinking it should be fine, if a `tag_post` is created since the order will be `nil` and should be at the bottom of the list. As for deleting, it should still be working since there will just be gaps in the order but will be fixed if the user re-saves the order.

Just a note, I am putting the order in the `TagPost` which is the assoc table since post and tag has many to many association